util/links.go

```diff
@@ -36,7 +36,7 @@ var (
 
 	// Substitute a particular custom column value for this token.
 	// TODO: Handle start/end columns (for custom columns and build IDs).
-	CustomColumnRe = regexp.MustCompile(`<custom-\d+>`)
+	CustomColumnRe = regexp.MustCompile(`<custom-\w+>`)
```

Comment:
For consistency, I think we still want to support <custom-N> (e.g. `<custom-0>` for the value of the first custom column header), so I don't think this regex should change.

That means that for getting the column headers, you'll need to compare to the column headers in configuration. This should be available in Summarizer, for instance, through updateTab()'s `group` argument (which contains the TestGroup configuration with the custom column headers), which can be used in failingTestSummaries() to pull the custom column header info.